### PR TITLE
create intern() string method definition

### PR DESCRIPTION
**Pull request by redpepper-nag**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/strings/terms/intern/intern.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Syntax
+
+string.intern()
+
+## Example
+
+String st1 = 'hello';
+String st2 = new String('Hello');
+String st3 = st1.intern(); // returns string same as st2
```
Comment (SSwiniarski) [inline content/java/concepts/strings/terms/intern/intern.md:22]:
```suggestion
String st3 = st2.intern(); // returns string same as st2
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@redpepper-nag I did a first review and left several comments. I may have more after you make these changes. You also need to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=1760).

**Comment by KTom101** [comment]
@redpepper-nag I've noticed it's been some time since the 1st review. Is there anything I can help with?

**Comment by SSwiniarski** [comment]
@redpepper-nag, if we don't hear from you in a week we'll need to close this PR and re-open the issue so someone else can work on it.

**Comment by SSwiniarski** [comment]
@redpepper-nag we're closing this as abandoned.

**Pull request closed without merging**